cyences_app_for_splunk/appserver/static/cs_asset_intelligence.js

```diff
@@ -48,6 +48,7 @@ function(mvc){
             let sophos = results[3];
             let defender = results[4];
             let crowdstrike = results[5];
+            let kaspersky = results[6];
```

Comment:
Bro, kindly update the comment also on line no. 43
- // fields: (7) ["lansweeper", "qualys", "tenable", "sophos", "defender", "crowdstrike", "kaspersky"]